spec/realtime/delta.test.js

```diff
@@ -0,0 +1,65 @@
+"use strict";
+
+define(['shared_helper', 'vcdiff-decoder'], function(helper, vcdiffDecoder) {
+	var exports = {},
+		displayError = helper.displayError,
+		closeAndFinish = helper.closeAndFinish,
+		monitorConnection = helper.monitorConnection;
+
+	var equals = function(a, b) {
+		return JSON.stringify(a) === JSON.stringify(b);
+	};
+
+	exports.setupMessage = function(test) {
+		test.expect(1);
+		helper.setupApp(function(err) {
+			if(err) {
+				test.ok(false, displayError(err));
+			} else {
+				test.ok(true, 'setup app');
+			}
+			test.done();
+		});
+	};
+
+	exports.deltaCodec = function(test) {
+		test.expect(3);
+		var testName = 'delta-codec';
+		try {
+			var testData = [
+				{ foo: 'bar', count: 1, status: 'active' },
+				{ foo: 'bar', count: 2, status: 'active' },
+				{ foo: 'bar', count: 2, status: 'inactive' }
+			];
+
+			var realtime = helper.AblyRealtime({
+				plugins: {
+					'vcdiffDecoder': vcdiffDecoder
+				}
+			});
+			var channel = realtime.channels.get('[?delta=vcdiff]' + testName);
```

Comment:
Doesn't matter much, but since this is based on feature-channel-params, would be good to test with that

Reply:
The problem is channel params are not deployed on sandbox at this point of time since they are also a PR. Until they make it to sandbox this is the only way to have this test run.